PaymentButtons/src/main/java/com/paypal/android/paymentbuttons/PayPalButton.kt

```diff
@@ -112,13 +112,15 @@ open class PayPalButton @JvmOverloads constructor(
  *
  * @see GOLD is the default color if one is not provided and is the recommended choice as research
```

Comment:
Is gold the default or is another color the default now? We should also add a docstring like " Deprecated - please use BLUE, BLACK, or SILVER." to the gold and white docstrings.

Reply:
not sure, trying to find out.

Reply:
It doesn't seem like dark blue is an option for PayPalButton. Was gold the default for this button before?

Reply:
My bad. Dark blue was the default for PayPalCreditButton.

We had deprecated blue, black, and silver colors for PayPalButton. No changes were made to gold and white, including the comments.
So yes, gold was and is the default color for PayPal button.

Reply:
That is super helpful, thanks for the rundown of the current + previous status!